check_keywords.py

```diff
@@ -26,6 +27,14 @@
     check=True,
 ).stdout.split()
 
+vscode_names = []
+for item in json.load(open("./oomph-vscode/syntaxes/oomph.tmLanguage.json"))["repository"]["keywords"]["patterns"]:
+    pattern = item["match"]
+    pattern = pattern[pattern.find("(")+1:pattern.rfind(")")]
```

Comment:
`find()` isn't great, because it fails silently:

```suggestion
    pattern = item["match"].split(")")[0].split("(")[1]
```

Reply:
Why not utilize `str.index` instead of `str.split`?

Reply:
I'll accept that too.